README.md

```diff
@@ -85,46 +86,49 @@ Examples
 const puppeteer = require('puppeteer')
 const ScreenshotTester = require('puppeteer-screenshot-tester')
 
-describe('google test', () => {
+describe('example test', () => {
   let originalTimeout
 
   // extend default interval to 10s because some image processing might take some time
   // we can do it beforeEach or once per test suite it's up to you
   // if you're running that on fast computer/server probably won't need to do that
-  beforeEach(function() {
+  beforeEach(() => {
     originalTimeout = jasmine.DEFAULT_TIMEOUT_INTERVAL
-    jasmine.DEFAULT_TIMEOUT_INTERVAL = 10000
+    jasmine.DEFAULT_TIMEOUT_INTERVAL = 100000
   })
 
   // set default interval timeout for jasmine
-  afterEach(function() {
+  afterEach(() => {
     jasmine.DEFAULT_TIMEOUT_INTERVAL = originalTimeout
   })
 
-  it(`check if google exists`, async () => {
+  it(`check if example page looks the same`, async () => {
     // create ScreenshotTester with optional config
     const tester = await ScreenshotTester(0.8, false, false, [], {
       transparency: 0.5
     })
 
     // setting up puppeteer
-    const browser = await puppeteer.launch()
+    const browser = await puppeteer.launch({ headless: true })
     const page = await browser.newPage()
-    await page.setViewport({width: 1920, height: 1080})
-    await page.goto('https://www.google.com', { waitUntil: 'networkidle0' })
-    await page.type('input[title="Search"]', 'Hello', { delay: 100 })
+    await page.setViewport({ width: 1280, height: 720 })
+    await page.goto('https://example.com', { waitUntil: 'networkidle0' })
```

Comment:
Testing `google.com` is done on purpose, it's easier to inflict change on google.com than on example.com which is basically static website.